cypress/e2e/cloud/buckets.test.ts

```diff
@@ -16,24 +16,30 @@ describe('Buckets', () => {
 
   // TODO: Skipping this until we can sort out the differences between OSS and Cloud
   it('can sort by name and retention', () => {
-    const buckets = [
-      'Website Monitoring Bucket',
-      'defbuck',
-      '_tasks',
-      '_monitoring',
-    ]
-    const retentionDesc = [
-      '_monitoring',
-      'Website Monitoring Bucket',
-      '_tasks',
-      'defbuck',
-    ]
-    const retentionAsc = [
-      'defbuck',
-      '_tasks',
-      '_monitoring',
-      'Website Monitoring Bucket',
-    ]
+    const demoDataBucket = 'Website Monitoring Bucket'
+    const buckets = [demoDataBucket, 'defbuck', '_tasks', '_monitoring']
```

Comment:
Do you think it's worth using the global alias for the defaultbucket rather than the hard coded defbuck string?

Reply:
Ooo yes